index.bs

```diff
@@ -28,6 +28,72 @@ Blah blah blah.
 
 This specification depends on the Infra standard. [[!INFRA]]
 
+<h2 id="deployed-mitigations">Deployed Mitigations</h2>
+
+Some browsers have deployed and announced protections against
+navigational-tracking. This section is a work in progress to detail what
```

Comment:
I think we can omit the hyphen when talking about navigational tracking as a noun. I added it to the document's title because the compound noun is used as an adjective there to modify "mitigations". https://www.capstoneediting.com.au/blog/how-to-hyphenate-a-compound-noun#:~:text=adjective%20%2B%20noun
```suggestion
navigational tracking. This section is a work in progress to detail what
```